actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -265,7 +265,12 @@ def to_unsafe_h
     # the same way as <tt>Hash#each_pair</tt>.
     def each_pair(&block)
       @parameters.each_pair do |key, value|
-        yield key, convert_hashes_to_parameters(key, value)
+        converted_value = convert_hashes_to_parameters(key, value)
+        if block.arity == 1
```

Comment:
We can always yield an array since the block will rake care of destructing the array in two argument.

Reply:
Ah neat. Thanks, updated and rebased the PR.